k8sclient/k8sclient.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/intel/multus-cni/kubeletclient"
 	"github.com/intel/multus-cni/logging"
 	"github.com/intel/multus-cni/types"
+	nettypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```

Comment:
Can we use 'github.com/K8sNetworkPlumbingWG/network-attachment-definition-client' instead of 'github.com/k8snetworkplumbingwg/network-attachment-definition-client/'?

Reply:
will fix, thanks